static/js/src/ua-payment-modal.js

```diff
@@ -401,14 +401,21 @@ function applyLoggedInPurchaseTotals() {
           currentTransaction.accountId,
           currentTransaction.products,
           currentTransaction.previousPurchaseId
-        ).then((purchasePreview) => {
-          currentTransaction.total = purchasePreview.total;
-          currentTransaction.tax = purchasePreview.taxAmount;
-          modal.classList.remove("is-processing");
-          setOrderTotals(country, vatApplicable, purchasePreview, modal);
-        });
+        )
+          .then((purchasePreview) => {
+            currentTransaction.total = purchasePreview.total;
+            currentTransaction.tax = purchasePreview.taxAmount;
+            modal.classList.remove("is-processing");
+            setOrderTotals(country, vatApplicable, purchasePreview, modal);
+          })
+          .catch((error) => {
+            modal.classList.remove("is-processing");
+            setOrderTotals(null, false, currentTransaction, modal);
+            console.error(error);
```

Comment:
Can you report this issue to sentry?

Reply:
We do catch those in the back-end already. It might be redundant to add them in the front-end too.